components/component_style/form.css

```diff
@@ -0,0 +1,33 @@
+.form-link {
+  display: flex;
+  align-items: center;
+  gap: var(--default-spacing);
+  justify-content: center;
+}
+
+.form-container {
+  margin-right: auto;
+  margin-left: auto;
+  max-width: 31.25rem;
+  flex-basis: 95%;
+  background-color: var(--basic-bg);
+  padding-bottom: var(--default-spacing);
+  box-shadow: var(--default-shadow);
+  border-radius: var(--default-border);
+}
+
+.form-heading {
+  text-align: center;
+  padding: var(--default-spacing) 0;
+}
+
+.form-extrafeild {
+    display: flex;
+    justify-content: space-between;
+    padding-right: var(--default-spacing);
+}
+
+.btn-extra {
+    width: -webkit-fill-available;
```

Comment:
Using `-webkit-fill-available` will only make it work with Webkit-based browsers. Why don't you mention `width: 100%` instead

Reply:
problem with `width: 100%`
![image](https://user-images.githubusercontent.com/72219676/154759060-89e5a178-9d81-48f6-a63c-0a993a3c685c.png)

Reply:
In that case, I suggest you add `width: -moz-available` as well to support firefox browsers

Reply:
thanks for your input, have made the changes